docs/release-notes/2.33.0.md

```diff
@@ -0,0 +1,160 @@
+
+- [PMM-10881](https://jira.percona.com/browse/PMM-10881): [CVE]: As part of PMM 2.33.0, grafana was updated to 9.2.5, which fixes some critical vulnerabilities. For more information, see [grafana 9.2.5](https://grafana.com/blog/2022/11/08/security-release-new-versions-of-grafana-with-critical-and-moderate-fixes-for-cve-2022-39328-cve-2022-39307-and-cve-2022-39306/).
+
+
+- [PMM-10135](https://jira.percona.com/browse/PMM-10135): Due to its limited use, the **Examples** tab has been disabled in PMM 2.33.0. Also, to safeguard our user's privacy, we have removed Personally Identifiable Information (PII) from the **Explain** tab.
+
+
+- [PMM-8655](https://jira.percona.com/browse/PMM-8655): Latest state of agents is now available on the PMM server after the connection is re-established between the client and server.
+
+- [PMM-10969](https://jira.percona.com/browse/PMM-10969): VictoriaMetrics operator has been upgraded from 0.24 to 0.29.0.
+
+
+- [PMM-10554](https://jira.percona.com/browse/PMM-10554): Postgres collector now provides runtime duration for scrapes at the collector level. As a result, we can identify collectors who take too long. By utilizing this data, corrective action can be taken.
+
+
+
+- [PMM-10629](https://jira.percona.com/browse/PMM-10629): [Components Upgrade]: VictoriaMetrics has been upgraded from 1.77.1 to 1.82.1.
+
+
+
+
+## Bugs Fixed
+
+- [PMM-10858](https://jira.percona.com/browse/PMM-10858): In specific environments, including AWS, some EPEL repository mirrors did not respond within the time limit defined by `pmm-update`, currently set to 30 seconds. This was causing `supervisord` to kill `pmm-update-checker`, the component responsible for verifying if a newer version of the PMM Server is available for upgrade. This issue has been resolved now.
+
+
+- [PMM-10683](https://jira.percona.com/browse/PMM-10683): [DBaaS]: While registering a K8s cluster on the **Manage Components Versions** modal, the versions were not getting unchecked. The issue has now been resolved.
+
+
+- [PMM-10637](https://jira.percona.com/browse/PMM-10637): Fixed an issue where the queries for some panels on the PostgreSQL Instance Summary dashboards were broken.
+
+
+- [PMM-10732](https://jira.percona.com/browse/PMM-10732): Checking the select all checkbox on the **Services**, **Agents**, and **Nodes** modal in the **PMM Inventory** window deleted all entries on all the pages. The issue has been resolved now.
+
+
+- [PMM-10983](https://jira.percona.com/browse/PMM-10983): Some panels on the dashboards were showing only short names. The issue has been resolved now.
+
+
+- [PMM-10996](https://jira.percona.com/browse/PMM-10996): [Backup Management]: Fixed an issue where the initial description text for a storage location could not be updated.
+
+- [PMM-11029](https://jira.percona.com/browse/PMM-11029): There was typo on the Home dashboard: **COMMAND CENTER** graphs. This has been fixed now.
```

Comment:
Is this number correct? Because https://jira.percona.com/browse/PMM-11029 its related to bug in QAN.

Reply:
Fixed the number. Thanks for pointing out.